### ISSUE TITLE
aci_spanning_tree_interface_policy doesn't apply control attributes

### ISSUE THREAD

**Issue by scotttyso** [CiscoDevNet/terraform-provider-aci#444]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

0.15.4

### APIC version and APIC Platform

* V x.x.x and on-prem/cloud-aws/cloud-azure/all.
* 5.1(4c)

### Affected Resource(s)

* aci_spanning_tree_interface_policy

### Terraform Configuration Files

```hcl
resource "aci_spanning_tree_interface_policy" "STP_bpduFG" {
    description = "BPDU Filter and Guard"
    name        = "example"
    ctrl        = ["bpdu-filter","bpdu-guard"]
}

resource "aci_spanning_tree_interface_policy" "STP_bpduFt" {
    description = "BPDU Filter"
    name        = "example"
    ctrl        = ["bpdu-filter"]
}

resource "aci_spanning_tree_interface_policy" "STP_bpduGd" {
    description = "BPDU Guard"
    name        = "example"
    ctrl        = ["bpdu-guard"]
}

resource "aci_spanning_tree_interface_policy" "STP_default" {
    description = "Both Disabled"
    name        = "example"
    ctrl        = ["unspecified"]
}

```

### Debug Output

I have run the plan multiple times and each time it comes back saying the policy resources need to be updated.  In the APIC I see each of the policy objects created but the ctrl parameters are all blank on all policies above.

```
Note: Objects have changed outside of Terraform

Terraform detected the following changes made outside of Terraform since the last "terraform apply":

  # aci_spanning_tree_interface_policy.STP_bpduGd has been changed
  ~ resource "aci_spanning_tree_interface_policy" "STP_bpduGd" {
      ~ ctrl        = [
          - "bpdu-guard",
          + "bpdu-filter",
        ]
      ~ description = "BPDU Guard" -> "BPDU Filter"
        id          = "uni/infra/ifPol-example"
        name        = "example"
        # (1 unchanged attribute hidden)
    }
  # aci_spanning_tree_interface_policy.STP_default has been changed
  ~ resource "aci_spanning_tree_interface_policy" "STP_default" {
      ~ ctrl        = [
          - "unspecified",
          + "bpdu-filter",
        ]
      ~ description = "Both Disabled" -> "BPDU Filter"
        id          = "uni/infra/ifPol-example"
        name        = "example"
        # (1 unchanged attribute hidden)
    }
  # aci_spanning_tree_interface_policy.STP_bpduFG has been changed
  ~ resource "aci_spanning_tree_interface_policy" "STP_bpduFG" {
      ~ ctrl        = [
            "bpdu-filter",
          - "bpdu-guard",
        ]
      ~ description = "BPDU Filter and Guard" -> "BPDU Filter"
        id          = "uni/infra/ifPol-example"
        name        = "example"
        # (1 unchanged attribute hidden)
    }

Unless you have made equivalent changes to your configuration, or ignored the relevant attributes using ignore_changes, the following plan may include actions to undo or respond to these changes.

──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # aci_spanning_tree_interface_policy.STP_bpduFG will be updated in-place
  ~ resource "aci_spanning_tree_interface_policy" "STP_bpduFG" {
      ~ ctrl        = [
            "bpdu-filter",
          + "bpdu-guard",
        ]
      ~ description = "BPDU Filter" -> "BPDU Filter and Guard"
        id          = "uni/infra/ifPol-example"
        name        = "example"
        # (1 unchanged attribute hidden)
    }

  # aci_spanning_tree_interface_policy.STP_bpduGd will be updated in-place
  ~ resource "aci_spanning_tree_interface_policy" "STP_bpduGd" {
      ~ ctrl        = [
          - "bpdu-filter",
          + "bpdu-guard",
        ]
      ~ description = "BPDU Filter" -> "BPDU Guard"
        id          = "uni/infra/ifPol-example"
        name        = "example"
        # (1 unchanged attribute hidden)
    }

  # aci_spanning_tree_interface_policy.STP_default will be updated in-place
  ~ resource "aci_spanning_tree_interface_policy" "STP_default" {
      ~ ctrl        = [
          - "bpdu-filter",
          + "unspecified",
        ]
      ~ description = "BPDU Filter" -> "Both Disabled"
        id          = "uni/infra/ifPol-example"
        name        = "example"
        # (1 unchanged attribute hidden)
    }
```

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by lhercot** [comment]
Could you retry with latest 0.7.1 version and let me know if this is still an issue?

**Comment by lhercot** [comment]
Checked with @scotttyso via PM and issue is resolved in v0.7.1
